proteus/Domain.py

```diff
@@ -301,6 +302,45 @@ def writeAsymptote(self, fileprefix):
     def writeXdmf(self,ar):
         raise UserWarning("Xdmf output not implemented")
 
+def unitSimplex(nd=2):
+    """Builds a 2 or 3 dimension reference element.
+    
+    Parameters
+    ----------
+    nd : int
+        The elements dimension (must be 2 or 3)
+
+    Returns
+    -------
+    reference_element
+    """
+    if nd!=2 and nd!=3:
+        logEvent("ERROR - Reference element must have dimension 2 or 3")
+        sys.exit(1)
```

Comment:
So I think we should make a decision about this.  I'm following a method I saw @adimako use, but if we'd rather use built in exceptions I'm open to that too.  I think we should be consistent throughout the library with whatever we choose to use.  @nerisaurus?